cfme/tests/infrastructure/test_webmks_console.py

```diff
@@ -79,74 +78,51 @@ def test_webmks_vm_console(request, appliance, provider, vm_obj,
 
     request.addfinalizer(vm_console.close_console_window)
     request.addfinalizer(appliance.server.logout)
+    try:
```

Comment:
**REQUIRED**
Too big "try except" block, use it where it's really required.

Reply:
As discussed on IRC, kindly resolve this.

Reply:
@kedark3 can you summarize the discussion and why it was determined this is acceptable here?

Reply:
@mshriver : try catch block is put that way because if test fails, any of the assertion goes wrong, I want to take the screenshot. I did not use finalizer because I do not need screenshot if test passes and hence except is generic and it will re-raise the original exception after the screenshot is taken. Hence @quarckster  approved it with suggestion to as exception "Exception" on line 132.